openerp/addons/base/ir/ir_qweb.py

```diff
@@ -432,6 +432,19 @@ def render_tag_if(self, element, template_attributes, generated_attributes, qweb
 
     def render_tag_call(self, element, template_attributes, generated_attributes, qwebcontext):
         d = qwebcontext.copy()
+
+        if 'lang' in template_attributes:
+            # we need to store the original language
+            init_lang = d.context.get('lang', 'en_us')
+            lang = template_attributes['lang']
+            # lang can be a lang code (ex: 'en_US' or 'fr') or a record field
+            is_empty_field = self.eval(lang, d) == False
```

Comment:
tricky bit: eval return false if the field exists but is empty, but returns nothing if you eval a string (which contains not '.')